packages/core/src/components/Chip.story.tsx

```diff
@@ -27,29 +58,24 @@ storiesOf('Core/Chip', module)
       </Spacing>
     </>
   ))
-  .add('With an icon.', () => <Chip icon={<IconCloseAlt decorative size="2em" />}>Chip</Chip>)
   .add('With an icon button.', () => (
     <>
       <Spacing right={1} inline>
-        <Chip icon={<IconCloseAlt decorative size="2em" />} onIconClick={action('onIconClick')}>
+        <Chip afterIcon={<IconCloseAlt size="2em" />} onIconClick={action('onIconClick')}>
```

Comment:
Why'd you remove `decorative` from all the icons? They will now log warnings to the console.